libs/transloco-locale/src/lib/transloco-locale.service.ts

```diff
@@ -36,19 +36,28 @@ export class TranslocoLocaleService implements OnDestroy {
   private numberTransformer = inject(TRANSLOCO_NUMBER_TRANSFORMER);
   private dateTransformer = inject(TRANSLOCO_DATE_TRANSFORMER);
   private localeConfig: LocaleConfig = inject(TRANSLOCO_LOCALE_CONFIG);
+  private browserLocale = navigator?.language || this.defaultLocale;
 
-  private _locale =
-    this.defaultLocale || this.toLocale(this.translocoService.getActiveLang());
+  private _locale = '';
   private locale: BehaviorSubject<Locale> = new BehaviorSubject(this._locale);
-  private subscription: Subscription | null = this.translocoService.langChanges$
-    .pipe(
-      map((lang) => this.toLocale(lang)),
-      filter(Boolean),
-    )
-    .subscribe((locale: Locale) => this.setLocale(locale));
+  private subscription: Subscription | null = null;
 
   localeChanges$ = this.locale.asObservable().pipe(distinctUntilChanged());
 
+  constructor() {
+    // Initialize locale with fallback chain
+    this._locale = this.initializeLocale();
+    this.locale.next(this._locale);
```

Comment:
Can be simplified to:
```suggestion
    // Initialize locale with fallback chain
    this.setLocale(this.initializeLocale());
```

Reply:
updated